Code/GraphMol/MolChemicalFeatures/Wrap/MolChemicalFeature.cpp

```diff
@@ -54,8 +54,13 @@ struct feat_wrapper {
         .def("GetPos",
              (RDGeom::Point3D(MolChemicalFeature::*)(int) const) &
                  MolChemicalFeature::getPos,
-             (python::arg("self"), python::arg("confId") = -1),
+             (python::arg("self"), python::arg("confId")),
              "Get the location of the chemical feature")
+        .def("GetPosDefault",
```

Comment:
I'm curious why you went the route of adding a new method name for this.

Given that you it would be possible to just have an an overload of `GetPos()`, and that this change breaks existing code, I wonder why you chose to go this way.

Reply:
You are correct. I changed it.